_unit-test/_test_setup.sh

```diff
@@ -0,0 +1,52 @@
+set -euo pipefail
```

Comment:
Should this be removed from `install/`?

Reply:
all these duplicate files should be removed